src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -76,7 +93,13 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
         alt={`${userDisplayName}'s avatar`}
         onClick={onClick}
       />
-      {showStatus && <span className={statusIndicatorClasses} />}
+
+      {hasUserStatus && (
+        <span
+          className={statusIndicatorClasses}
+          style={statusIndicatorStyles}
+        />
+      )}
```

Comment:
Can you elaborate please, what exactly do you mean by tooltip? This is just the colorful dot that visualizes the status.

Reply:
@denisdimitrov  yeah, probably we should call @sparkletown/design @sparkletown/product here.
My concern is that the color of the status is not descriptive enough to understand what is the status of the other person. Let's say "sleeping" has a "pink" color but if I have never chosen "sleeping" status for myself, I won't be able to understand the meaning of the pink status. For that, I'd like to hover over the circle to get the description: what this colored status means - sleeping or dancing (for example).
But looks like in the current implementation we don't use the titles of the statuses besides the place where I edit my profile. 

Moreover, accessibility is really important for our clients. Distinguishing the statuses just by color is not enough for colorblind people, they will not be able to understand is this purple or green, for these purposes, it's always good to have some kind of tooltip.

like this for example (but display it only on hover)
![image](https://user-images.githubusercontent.com/63313732/124787835-d02fcc00-df16-11eb-855f-e7e3403d7e23.png)

Reply:
@denisdimitrov  I've just opened the issue. We have statuses in the designs. So you can just add them accordingly.
![image](https://user-images.githubusercontent.com/63313732/124788673-901d1900-df17-11eb-84c1-4ac6c460b77c.png)

![image](https://user-images.githubusercontent.com/63313732/124788693-97dcbd80-df17-11eb-8dc4-9a75b534f971.png)

because I feel that we miss that information.

Reply:
I love this suggestion, Viktoryia. However, considering deadlines are well past due, can we please get this merged if it's working properly, and then create a new PR for the optimization / accessibility upgrade you suggest? Right now we don't even have a way to add or test any statuses at all.

cc: @jonboutelle

Reply:
Yes please follow design above. That is what we are expecting to see. No tooltip necessary. Just put in the actual profile as text. Let's get this done, thanks!!!

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/6db8a73718475d1f8e780610acf3becd6f5e4c3b

Reply:
Is this an already existing issue, if not do you want me to backlog it?

Reply:
you mean statuses text? I took the screenshots from the issue you're going to close. 🙃 
https://github.com/sparkletown/internal-sparkle-issues/issues/720